nextflow.config

```diff
@@ -1,28 +1,13 @@
-
-singularity {
-    enabled = true
-    autoMounts = true
-    cacheDir = "/hps/nobackup/rdf/metagenomics/service-team/singularity-cache/"
-}
-
-profiles {
-    
-    standard { 
-            includeConfig 'configs/local.config'
-        }
-
-    slurm {
-            includeConfig "configs/slurm.config"
-            process.executor = "slurm"
-        }
-}
-
 params {
 
     outdir = "/hps/nobackup/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_v6"
     dummy_fasta = "/hps/software/users/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_nf/assets/dummy.fasta"
     empty_file = "/hps/software/users/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_nf/assets/EMPTY.txt"
     pipeline_version = "6"
+    publish_dir_mode = 'copy'
+
+    generate_amplicon_db = false 
+    generate_uniref90_db = true
```

Comment:
Shouldn't both be "false"?

Reply:
Yes, probably. I've set it like this because it was more convenient for me when I did test runs. 
Changed in commit [change generate_uniref90_db default value](https://github.com/EBI-Metagenomics/taxdb_generation_nf/pull/1/commits/88fcb7d2c76a00e678c76955cee62d4676933c54)